iocore/dns/I_DNSProcessor.h

```diff
@@ -28,11 +28,12 @@
 const int DOMAIN_SERVICE_PORT        = NAMESERVER_PORT;
 const int DEFAULT_DOMAIN_NAME_SERVER = 0;
 
-const int MAX_DNS_PACKET_LEN = 8192;
-const int DNS_RR_MAX_COUNT   = (MAX_DNS_PACKET_LEN - HFIXEDSZ + RRFIXEDSZ - 1) / RRFIXEDSZ;
-const int DNS_MAX_ALIASES    = DNS_RR_MAX_COUNT;
-const int DNS_MAX_ADDRS      = DNS_RR_MAX_COUNT;
-const int DNS_HOSTBUF_SIZE   = MAX_DNS_PACKET_LEN;
+const int MAX_DNS_REQUEST_LEN  = NS_PACKETSZ;
+const int MAX_DNS_RESPONSE_LEN = NS_MAXMSG + 1;
```

Comment:
I might be missing something, I got below error on macOS
```
In file included from /Users/masaori/src/github.com/apache/trafficserver-asf-master/iocore/dns/P_DNSConnection.h:34:
/Users/masaori/src/github.com/apache/trafficserver-asf-master/iocore/dns/I_DNSProcessor.h:32:34: error: use of undeclared identifier 'NS_MAXMSG'
const int MAX_DNS_RESPONSE_LEN = NS_MAXMSG + 1;
                                 ^
```

Reply:
Right it's not defined in `arpa/nameser.h` on macOS. I could buld latest commit.